Jenkinsfile

```diff
@@ -0,0 +1,16 @@
+node {
+    stage 'Checkout'
+    checkout scm
+
+    stage 'Build virtualenv'
+    sh '''virtualenv venv
+    . ./venv/bin/activate
+    pip install --upgrade pip --no-cache-dir
```

Comment:
Pip hangs if I try to install requirements or update pip. There seems to be an bug in combination with the cache-dir. If I don't use --no-cache-dir then it will hang forever.